src/main/java/org/reaktivity/nukleus/http2/internal/routable/Target.java

```diff
@@ -216,6 +225,72 @@ public void doHttpEnd(
         streamsBuffer.write(end.typeId(), end.buffer(), end.offset(), end.sizeof());
     }
 
+    public void doSettings(
+            long targetId,
+            int maxConcurrentStreams)
+    {
+        DataFW data = dataRW.wrap(writeBuffer, 0, writeBuffer.capacity())
+                            .streamId(targetId)
+                            .payload(p -> p.set(visitSettings(maxConcurrentStreams)))
+                            .extension(e -> e.reset())
+                            .build();
+
+        streamsBuffer.write(data.typeId(), data.buffer(), data.offset(), data.sizeof());
+    }
+
+    public void doSettingsAck(
+            long targetId)
+    {
+        DataFW data = dataRW.wrap(writeBuffer, 0, writeBuffer.capacity())
+                            .streamId(targetId)
+                            .payload(p -> p.set(visitSettingsAck()))
+                            .extension(e -> e.reset())
```

Comment:
Is it still necessary to explicitly zero out the extension, or is that handled automatically now?